docs/src/components/component/button/button.config.yml

```diff
@@ -0,0 +1,125 @@
+title: button
+collated: true
+variants:
+  - name: default
+    context:
+      isDefault: true
+      isOutline: false
+      isLink: false
+      buttonStatus: primary
+      buttonText: default
+  - name: base
+    context:
+      isDefault: true
+      isOutline: false
+      isLink: false
+      buttonStatus: base
+      buttonText: base
+  - name: warning
+    context:
+      isDefault: true
+      isOutline: false
+      isLink: false
+      buttonStatus: warning
+      buttonText: warning
+  - name: active
+    context:
+      isDefault: true
+      isOutline: false
+      isLink: false
+      buttonStatus: primary
+      buttonText: active
+  - name: disabled
+    context:
+      isDefault: true
+      isOutline: false
+      isLink: false
+      buttonStatus: cautious
+      buttonText: disabled
+  - name: finish
+    context:
+      isDefault: true
+      isOutline: false
+      isLink: false
+      buttonStatus: finish
+      buttonText: finish-outline
```

Comment:
I feel like this text isn't a ton of value. I would prefer to have real-ish text in here. Or if you really want this kind of text, maybe we can create it from the other variables without having a dedicated one.